tests/jax/systems/mappo_system_test.py

```diff
@@ -15,16 +20,14 @@
 
 @pytest.fixture
 def test_full_system() -> System:
-    """Add description here."""
+    """Full mava system fixture for testing"""
     return mappo.MAPPOSystem()
 
 
-# TODO: fix test
-@pytest.mark.skip(reason="test is currently breaking ci pipeline")
-def test_except_trainer(
+def test_mappo(
```

Comment:
I get that it is arbitrary which env/ network... we select but is the goal to have a single integration test for each system (and its variations) with the simplest network on the simplest env? If that is the case, then this name is fine (tf was `test_mappo_on_debugging_env`)

Reply:
DIscussed in meetings: goal is to use as simple param and env as possible

Reply:
Discussed in meeting and decided that we want to test as simple param as possible